feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.component.html

```diff
@@ -0,0 +1,77 @@
+<ng-container *ngIf="orderEntry$ | async as orderEntry">
+  <ng-container *ngIf="isBundleBasedConfigurator(orderEntry)">
+    <ng-container *ngIf="numberOfLineItems$ | async as numberOfItems">
+      <div class="cx-number-items">
+        {{
+          'configurator.header.items' | cxTranslate: { count: numberOfItems }
+        }}
+      </div>
+
+      <a
+        role="link"
+        (click)="toggleItems()"
+        (keyup)="clickOnEnter($event)"
+        tabindex="0"
+      >
```

Comment:
We should not use `<a>` tag for elements that are not links (don't have href attribute / routerLink directive). Why:
- it's important for screan readers (cc @developpeurweb)
- we're adding unnecessary complexity to the component: UI-events bindings in HTML, and methods in TS

Instead, we should use `<button>` tag:
- because it's sematically correct - it triggers JS logic
- (it supports Enter and Space by default)

Reply:
Btw. there are more places with custom handling UI events, which is a code smell:
![obraz](https://user-images.githubusercontent.com/4001059/115696477-053e7080-a363-11eb-9535-5ae0bc13c967.png)

Reply:
Moreover if something is actually a link (has href attribute or routerLink directive, but doesn't trigger JS logic), it should stay `<a>` tag. But we should NOT add custom support for activating it with an Enter key (becasue Enter it not supported for `<a>` tags natively). And we're fine with it! We'd rather avoid unnecessary logic in TS compoents. cc @developpeurweb

Reply:
@Larisa-Staroverova : can you check this?